### PR TITLE
Miscellaneous typescript def fixes: vtkSelectionNode, vtkMapper, vtkColorTransferFunction, vtkOpenGLRenderWindow

### PR DESCRIPTION
**Pull request by Julesdevops**
### Context
Some typescript definitions are incorrect.

### Results
Fixes some incorrect definitions.

### Changes
- [x] Documentation and TypeScript definitions were updated to match those changes

### PR and Code Checklist

- [x] [semantic-release](https://github.com/semantic-release/semantic-release) commit messages
- [x] Run `npm run reformat` to have correctly formatted code

### Testing

- [ ] This change adds or fixes unit tests 
- [ ] Tested environment:
  - **vtk.js**: 
  - **OS**: 
  - **Browser**:

### CODE REVIEW

**Comment by Julesdevops** [comment]
fyi @daker @bruyeret

File: Sources/Rendering/Core/ColorTransferFunction/index.d.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import vtkDataArray from '../../../Common/Core/DataArray';
-import { vtkObject } from '../../../interfaces';
+import vtkScalarsToColors from '../../../Common/Core/ScalarsToColors'
 import { ColorSpace, Scale } from "./Constants";
 
 
 /* TODO: use VtkScalarsToColors instead of VtkObject */
-export interface vtkColorTransferFunction extends vtkObject {
+export interface vtkColorTransferFunction extends vtkScalarsToColors {
```
Comment (daker) [inline Sources/Rendering/Core/ColorTransferFunction/index.d.ts:7]: remove the TODO?

**Review by daker — CHANGES_REQUESTED** [review]

File: Sources/Rendering/Core/ColorTransferFunction/index.d.ts (hunk 1 above)
Comment (Julesdevops) [reply Sources/Rendering/Core/ColorTransferFunction/index.d.ts:6]: oops, how did I missed that? just fixed it

**Comment by finetjul** [comment]
image

**Comment by floryst** [comment]
@Julesdevops can you also add a commit to fix the typing issue from https://github.com/Kitware/vtk-js/issues/2705?

**Comment by Julesdevops** [comment]
No problem @floryst 
Actually I'm looking for a good way of getting rid of the typescript build errors we get. I think we'll also get one from adding `setBackground` to `vtkRenderer` definitions. since the way we extends those functions is a big nono for typescript

**Comment by floryst** [comment]
@Julesdevops Just checking in on this. I can help with this if need be.

**Comment by Julesdevops** [comment]
Thanks @floryst yeah did not have time to look into it lately but hopefully just fixed the typescript definitions check.
wdyt of this now?

**Comment by floryst** [comment]
LGTM!

**Review by daker — APPROVED** [review]
LGTM

**Pull request merged**